etl/steps/data/garden/regions/2023-01-01/migration.py

```diff
@@ -0,0 +1,884 @@
+# they should already be counted within another country.
+# This list is mainlly used to create aggregates (i.e. to add up data from different countries to obtain data for
+# continents).
+continent_members = {
+    "Africa": [
+        "Algeria",
+        "Angola",
+        "Benin",
+        "Botswana",
+        "Burkina Faso",
+        "Burundi",
+        "Cameroon",
+        "Cape Verde",
+        "Central African Republic",
+        "Chad",
+        "Comoros",
+        "Congo",
+        "Cote d'Ivoire",
+        "Democratic Republic of Congo",
+        "Djibouti",
+        "Egypt",
+        "Equatorial Guinea",
+        "Eritrea",
+        "Eritrea and Ethiopia",
+        "Eswatini",
+        "Ethiopia",
+        "Gabon",
+        "Gambia",
+        "Ghana",
+        "Guinea",
+        "Guinea-Bissau",
+        "Kenya",
+        "Lesotho",
+        "Liberia",
+        "Libya",
+        "Madagascar",
+        "Malawi",
+        "Mali",
+        "Mauritania",
+        "Mauritius",
+        "Mayotte",
+        "Morocco",
+        "Mozambique",
+        "Namibia",
+        "Niger",
+        "Nigeria",
+        "Reunion",
+        "Rwanda",
+        "Saint Helena",
+        "Sao Tome and Principe",
+        "Senegal",
+        "Seychelles",
+        "Sierra Leone",
+        "Somalia",
+        # 'Somaliland',
+        "South Africa",
+        "South Sudan",
+        "Sudan",
+        "Tanzania",
+        "Togo",
+        "Tunisia",
+        "Uganda",
+        "Western Sahara",
+        "Zambia",
+        # 'Zanzibar',
+        "Zimbabwe",
+    ],
+    "Asia": [
+        # 'Abkhazia',
+        "Afghanistan",
+        # 'Akrotiri and Dhekelia',
+        "Armenia",
+        "Azerbaijan",
+        "Bahrain",
+        "Bangladesh",
+        "Bhutan",
+        "British Indian Ocean Territory",
+        "Brunei",
+        "Cambodia",
+        "China",
+        "Christmas Island",
+        "Cocos Islands",
+        "East Timor",
+        "Georgia",
+        "Hong Kong",
+        "India",
+        "Indonesia",
+        "Iran",
+        "Iraq",
+        "Israel",
+        "Japan",
+        "Jordan",
+        "Kazakhstan",
+        "Kuwait",
+        "Kyrgyzstan",
+        "Laos",
+        "Lebanon",
+        "Macao",
+        "Malaysia",
+        "Maldives",
+        "Mongolia",
+        "Myanmar",
+        # 'Nagorno-Karabakh',
+        "Nepal",
+        "North Korea",
+        # 'Northern Cyprus',
+        "Oman",
+        "Pakistan",
+        "Palestine",
+        "Philippines",
+        "Qatar",
+        "Republic of Vietnam",
+        "Saudi Arabia",
+        "Singapore",
+        "South Korea",
+        # 'South Ossetia',
+        "Sri Lanka",
+        "Syria",
+        "Taiwan",
+        "Tajikistan",
+        "Thailand",
+        # 'Timor',
+        "Turkey",
+        "Turkmenistan",
+        "United Arab Emirates",
+        "United Korea",
```

Comment:
Is this the official name for the country before it split into North and South? I thought it was more commonly known as just 'Korea'.

Reply:
Yes, again, I don't know who introduced this definition in our dataset, but `United Korea` refers to the country before the split. I suppose it was called `United Korea` to avoid confusion. Feel free to propose a better name, e.g. `Korea (former)`.

Reply:
In this case, I would not use '(former)', as some may understand that as there being a 'Korea' in the present. I would simply use 'Korea', which is the name usually used.